string action
string palletizer
string robot
string node
---
bool result
